examples/complete/main.tf

```diff
@@ -39,10 +39,6 @@ module "ecs" {
     FARGATE_SPOT = {
       weight = 50
     }
-    ASG = {
```

Comment:
these other changes to the example were because they were now showing errors when run today 🤷🏽